.travis.yml

```diff
@@ -36,14 +25,34 @@ before_install:
   - export PATH=$HOME/.yarn/bin:$PATH
 
 install:
-  - yarn install --no-lockfile --non-interactive
-
-before_script:
-  - $BEFORE_SCRIPT
-
-script:
-  - yarn test:node
-  - yarn $SCRIPT
-
-after_script:
-  - $AFTER_SCRIPT
+  - yarn install --non-interactive
+
+jobs:
+  fail_fast: true
+
+  include:
+    # runs tests with current locked deps and linting
+    - env: NAME=test # used only to make Travis UI show description
```

Comment:
Lets have this be the main `test` stage, and then make the other things be `additional tests` stage (no need to run them if we don't have non-floating chrome only tests passing)...